Game/Source/GDKTestGyms/BenchmarkGymGameModeBase.cpp

```diff
@@ -486,22 +481,27 @@ void ABenchmarkGymGameModeBase::TickUXMetricCheck(float DeltaSeconds)
 
 void ABenchmarkGymGameModeBase::TickActorCountCheck(float DeltaSeconds)
 {
-	const UNFRConstants* Constants = UNFRConstants::Get(GetWorld());
-	check(Constants);
-	FMetricTimer TempActorCheckDelay = Constants->ActorCheckDelay;
 	// This test respects the initial delay timer in both native and GDK
-	if (TempActorCheckDelay.HasTimerGoneOff() &&
-		!TestLifetimeTimer.HasTimerGoneOff())
+	if (TickActorCountTimer.HasTimerGoneOff())
 	{
 		TryInitialiseExpectedActorCounts();
-		
-		const UWorld* World = GetWorld();
-		const FString WorkerID = GetGameInstance()->GetSpatialWorkerId();
-		const int32 ActualCountSimulate = GetActorClassCount(ExpectedSimPlayersCount.ActorClass);
-		const int32 ActualCountNPCs = GetActorClassCount(ExpectedNPCsCount.ActorClass);
-		GenerateTotalNumsForActors(WorkerID, World, ExpectedSimPlayersCount, MapAuthoritativeSimPlayers, SmoothedTotalAuthSimPlayers, ActualCountSimulate, false);
-		ReportAuthoritativeNPCs(WorkerID, World, ActualCountNPCs);
-		TempActorCheckDelay.SetTimer(1);
+
+		if (HasAuthority())
+		{
+			ActorCountReportIdx++;
```

Comment:
I wonder if clearing the count map here would make it easier to track count updates not coming back (stress scenarios?)

Reply:
or setting them to 0, and letting that fail them in 10 seconds time